libs/feature/editor/src/lib/models/editor-config.model.ts

```diff
@@ -17,6 +17,7 @@ export interface FormFieldConfig {
 }
 
 export interface EditorField {
+  id?: string
```

Comment:
This is not needed anymore, since you used the facade to retrieve multiple fields.